src/cloud/docker/docker-extend.md

```diff
@@ -75,6 +75,76 @@ To add a new service version using a `Dockerfile`:
 
 1. Once the build succeeds, test the changes by specifying the [Docker build sources].
 
+## Add a new PHP extension
+
+In addition to built in extensions, you can add PHP extensions to the PHP container.
+
+{:.procedure}
+To add a new PHP extension:
+
+1. Clone the `{{site.data.var.mcd}}` project to your local environment if necessary.
+
+1. On the command line, navigate to the following directory
+   ```bash
+      cd magento-cloud-docker/src/Compose/Php
+   ```
+
+1. Open `ExtensionResolver.php` file, define the required extension inside the `getConfig` method by specifying the extension type and dependency.
```

Comment:
```suggestion
1. Open the `ExtensionResolver.php` file, define the required extension inside the `getConfig` method by specifying the extension type and dependency.
```

It seems to me that after clone,  it may be better for the user to change to the project directory. Then this step would be to

- Open the `ExtensionResolver.php` file in the `magento-cloud-docker/src/Compose/Php` directory...

Or maybe 

- In an editor, open the `magento-cloud-docker/src/Compose/Php/ExtensionResolver.php` file, and ...

The reasoning is, if you make them change to the deeply embedded folder, then they have to change directory again later. This way they can call that file in their editor without changing the directory, perhaps? Anyway, it's not a big deal, just a matter of which path to take the user and remembering the state that you left them in. Later, they have to call an ece-tools command and my question would be, should I still be in the `magento-cloud-docker/src/Compose/Php` directory?